src/main/java/com/example/sharemind/counselor/application/CounselorServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.example.sharemind.counselor.application;
+
+import com.example.sharemind.counselor.domain.Counselor;
+import com.example.sharemind.counselor.exception.CounselorErrorCode;
+import com.example.sharemind.counselor.exception.CounselorException;
+import com.example.sharemind.counselor.repository.CounselorRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class CounselorServiceImpl implements CounselorService {
+
+    private final CounselorRepository counselorRepository;
+
+    @Override
+    public String getCounselorNickname(Long counselorId) {
```

Comment:
닉네임까지 구체적으로 넘겨주지 말고 getCounselor로 두고 Counselor를 넘겨주는 건 어떨까요!? StompPreHandler에서는 counselor.getNickname()으로 쓰는 방식으로요...!
counselorId로 Counselor 찾는 로직은 StompPreHandler 외에 다른 서비스 레이어들에서도 사용될 수 있을 것 같아 그렇게 하면 사용성(?)이 높아질 수 있을 것 같습니다 (당장 ConsultServiceImpl에서도 활용될 수 있겠네요)
```suggestion
    public Counselor getCounselorByCounselorId(Long counselorId) {
```